src/index.js

```diff
@@ -79,4 +85,13 @@ client.commands = new Collection();
 
 registerGlobalErrorHandlers(client);
 
+process.on('uncaughtException', (error) => {
+  handleCriticalError(error, client);
```

Comment:
actually you're not using this client that you're passing as second parameter, so you can remove it from here

```suggestion
    handleCriticalError(error);
```